### DISCUSSION TITLE
Unable to fork and clone

### DISCUSSION THREAD

**Discussion by ragini-gp** [suryanshsk/Python-Voice-Assistant-Suryanshsk#233] [General]
@suryanshsk 
I am unable to clone the forked repo in my local
I encountered a problem while attempting to clone it : The process fails due to the following invalid path:
Rhythm of Knowledge: Get It Right, Hear It Right!

**Comment by suryanshsk** [comment]
clone the actual link

**Comment by suryanshsk** [comment]
or first install git

**Reply by ragini-gp** [reply]
I have already installed.
Also, I had contributed twice to this project already.
But, to get the latest updates, I deleted the forked repo, and tried to fork again and clone it. But could not.

**Reply by Aryan-9488** [reply]
hey ,I am now also getting the same issue tried different things but couldn't fix it.
How did you resolved it ??

**Reply by ragini-gp** [reply]
Just try again, now I am not getting error

**Comment by suryanshsk** [comment]
oh i think one last option is download this github file and then work on it
